include/seqan3/range/container/concatenated_sequences.hpp

```diff
@@ -198,23 +221,32 @@ class concatenated_sequences
     }
     //!\endcond
 
-    //!\brief Whether a type is compatible with this class's `value_type` or `reference` type.
-    //!\hideinitializer
+    /*!\brief Whether a type is compatible with this class's `value_type` or `reference` type.
+     * \hideinitializer
+     * \details
+     * \noapi{Exposition only.}
+     */
     // we explicitly check same-ness, because these types may not be fully resolved, yet
     template <std::ranges::range t>
     static constexpr bool is_compatible_with_value_type = is_compatible_with_value_type_aux(std::type_identity<t>{});
 
-    //!\brief Whether a type is compatible with this class.
-    //!\hideinitializer
+    /*!\brief Whether a type is compatible with this class.
+     * \hideinitializer
+     * \details
+     * \noapi{Exposition only.}
+     */
     // cannot use the concept, because this class is not yet fully defined
```

Comment:
Why is this not a part of the comment block above?

Reply:
This is more a message to the developer.

We could wrap that into `\if DEV`.